hugo/content/tutorials/customizing_cli.md

```diff
@@ -109,7 +109,7 @@ test()
 We'll save this under our project root as **test.logo**, and we can test that it's correct using our CLI like so:
```

Comment:
@steve-hickman-epistimis could you update this sentence to indicate that the program is saved under the examples folder instead? It wouldn't also be bad to point out that it's okay to save it elsewhere, so long as you keep in mind where it is 😉 .